backend/app/views/spree/admin/shared/_tabs.html.erb

```diff
@@ -3,7 +3,9 @@
 <% end %>
 
 <% if can? :admin, Spree::Product %>
-  <%= tab *Spree::BackendConfiguration::PRODUCT_TABS, icon: 'th-large' %>
+  <%= tab *Spree::BackendConfiguration::PRODUCT_TABS, icon: 'th-large' do %>
+    <%- render partial: 'spree/admin/shared/product_sub_menu' %>
```

Comment:
@tvdeyen @graygilmore I agree that this would be nice, but for compatibility reasons they should stay the same.

The sub_menus are a common deface target, and this has largely left it intact. An existing deface override on the data-hook in `spree/admin/shared/product_sub_menu` will still work. I think eventually we will end up cleaning this up through some sort of programatically extensible menu, and I'd rather not make developers have to change their menu overrides twice.